jsk_perception/src/draw_labeled_rect.cpp

```diff
@@ -0,0 +1,147 @@
+/*********************************************************************
```

Comment:
@k-okada I think it is possible with some change about the published topics as recognition results.
### VGG for classification: 

- Recognition: http://jsk-docs.readthedocs.io/en/latest/jsk_recognition/doc/jsk_perception/nodes/vgg16_object_recognition.html
- Visualization: http://jsk-docs.readthedocs.io/en/latest/jsk_recognition/doc/jsk_perception/nodes/draw_classification_result.html

## FasterRCNN for detection

- Recognition & Visualization: http://jsk-docs.readthedocs.io/en/latest/jsk_recognition/doc/jsk_perception/nodes/fast_rcnn.html